lib/engine/stock_market.rb

```diff
@@ -55,15 +55,25 @@ def move_up(corporation)
       return move_right(corporation) if one_d?
```

Comment:
Many games override stock_market movement. They will also need to be looked at.

Reply:
Good to know, thank you. I'm starting to suspect this isn't actually a great first issue to have chosen.